src/app/libs/dashboards/infrastructure/tokens.ts

```diff
@@ -0,0 +1,18 @@
+import { SettingsApi } from '../../settings/ui';
+import { DashboardsDataPort, DashboardsUiPort, DashboardsUseCase } from '../use-cases';
+import { DashboardsStoreService } from './store/dashboards-store.service';
+
+export const INFRASTRUCTURE_PROVIDERS = [
+  {
+    provide: DashboardsDataPort,
+    useExisting: DashboardsStoreService
+  }
+];
+
+export const APPLICATION_PROVIDERS = [
+  {
+    provide: DashboardsUiPort,
+    useExisting: DashboardsUseCase,
+    deps: [DashboardsDataPort, SettingsApi]
```

Comment:
Not sure about the dependency on the `SettingsApi` here as well. 🤔

Reply:
`SettingsApi` is definitely not needed here. We don't inject it in `DashboardsUseCase` constructor.